src/components/OpenMRSLogo.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const OpenMRSLogo: React.FC<React.DetailedHTMLProps<
+  React.HTMLAttributes<HTMLDivElement>,
+  HTMLDivElement
+  >> = () => {
+  return (
+      <img src="/openmrs_logo_white.gif"  alt={"openmrs"} style={{width:"30px"}} />
```

Comment:
Probably best to use a SVG rather than the GIF here. @suruchee: I've sent you an SVG of the OMRS logo via Slack.